recon/core/loader.py

```
@@ -0,0 +1,27 @@
+from typing import Callable
+from .registry import REGISTRY
+from .module import ReconModule
+
+
+def load_modules(package_root: str = "recon.modules") -> None:
+    """
+    Autodiscover modules under recon.modules.* that expose a callable get_module() -> ReconModule.
+    Example path: recon.modules.v1_osint.github_user
+    """
+    pkg = importlib.import_module(package_root)
+    for finder, name, ispkg in pkgutil.walk_packages(pkg.__path__, pkg.__name__ + "."):
```

Ruff (0.13.2)

Loop control variable `finder` not used within loop body

Rename unused `finder` to `_finder`

(B007)